src/Filter/Filter.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { MouseEventHandler, useState } from "react";
+import type { RadioChangeEvent } from "antd";
+import { Input, Modal, Button, Radio, Space } from "antd";
+import { DownOutlined } from "@ant-design/icons";
+import DateComp from "./DateComp";
+
+const { Search } = Input;
+const onSearch = (value: string) => console.log(value);
+
+const Filter = () => {
+  const [isModalVisible, setIsModalVisible] = useState(false);
+  const [value, setValue] = useState("Today");
+
+  const showModal = () => {
+    setIsModalVisible(true);
+  };
+
+  const handleCancel = () => {
+    setValue("Today");
```

Comment:
every time you cancel, it always goes back to "Today" ?
what if it was set to something different like "Last 7 days"

Reply:
Yes. When I click the cancel button it will display "Today".
I click the Apply button which option i clicked that value will be displayed.